lib/vquic/quiche.c

```diff
@@ -246,6 +251,10 @@ CURLcode Curl_quic_connect(struct connectdata *conn, curl_socket_t sockfd,
 
   result = flush_egress(conn, sockfd, qs);
   if(result) {
+    if(result == CURLE_AGAIN) {
+      /* Curl_quic_connect() should never returns CURLE_AGAIN */
```

Comment:
If it shouldn't, then how come it can? And why couldn't it? You have the same comment and code in `Curl_quic_is_connected()` too. The comment and logic is weird and I think it deserves more explanation!

Reply:
Ok, I have commented.